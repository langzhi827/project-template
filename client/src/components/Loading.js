/**
 *  Author: harry.lang
 *  Date: 2017/11/20
 *  Description: Created by harrylang on 2017/11/20.
 */
import React from 'react';

class Loading extends React.Component {
    render() {
        return <h1>正在加载......</h1>;
    }
}

export default Loading;