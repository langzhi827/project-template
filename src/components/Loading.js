/**
 *  Author: harry.lang
 *  Date: 2017/11/20
 *  Description: Created by harrylang on 2017/11/20.
 */
import ProxyComponent from 'ProxyComponent';

class Loading extends ProxyComponent {
    componentDidMount(){}
    render() {
        return <h1>正在加载......</h1>;
    }
}

export default Loading;