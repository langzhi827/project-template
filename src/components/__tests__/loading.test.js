import React from 'react';
import { render, mount } from 'enzyme';

import Loading from '../Loading';

describe('Loading', () => {
    it('renders correctly', () => {
        const wrapper = render(<Loading />);
        expect(wrapper).toMatchSnapshot();
    });

    it('mount correctly', () => {
        expect(() => mount(<Loading />)).not.toThrow();
    });
});