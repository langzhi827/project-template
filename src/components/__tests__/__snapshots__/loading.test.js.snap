// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading renders correctly 1`] = `
<h1>
  正在加载......
</h1>
`;
